prereise/gather/translate_zones.py

```diff
@@ -0,0 +1,229 @@
+import folium
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+from folium.features import DivIcon
+
+# IMPORTANT: rtree is required for overlay function, but not imported
+
+
+def plot_zone_map(gdf, interactive=False):
```

Comment:
The map looks great in the demo! Not sure whether we would like the plot parameters in this function to be hardcoded (size, color, etc.) or not, given we might want to look into a smaller area rather than the entire nation some day. Maybe this is future work.